hash techniques for orientation/DCT-basedHashforRobustness.py

```diff
@@ -0,0 +1,51 @@
+# Import dependencies
+from PIL import Image
+import imagehash
+import numpy as np
+import glob
+import sys
+import os
+import csv
+
+#input and output file paths
+input_path1 = r'/Users/rakshithanagendrappa/Desktop/NormalOrientationMaps/*.jpg'
```

Comment:
instead, make this  `input_dir` and the value would be the path `/Users/rakshithanagendrappa/Desktop/NormalOrientationMaps/` no need to use regex.

Use `os.listdir` instead:
https://www.tutorialspoint.com/python/os_listdir.htm

Reply:
Prof, os.listdir gets the list of files but the hash function takes the path to the file in order to perform hashing, hence chose to get file path instead of just files in a directory.

Reply:
Noted!

Reply:
Updated